models/modules/diffusion_generator.py

```diff
@@ -188,16 +207,25 @@ def p_mean_variance(
         y_cond=None,
         guidance_scale=0.0,
     ):
+        frame = 0
```

Comment:
Put these transforms in a function and call it here.

Reply:
a transform function is defined in diffusion_util.py and is called when needed